### DISCUSSION TITLE
Event Handlers for WinBox / Modal

### DISCUSSION THREAD

**Discussion by shawn-mcc** [EvotecIT/PSWriteHTML#452] [Q&A]
I've been using this module for a while now for automating some reports and I absolutely love it! Big shoutout to everyone involved. This is a silly question and I know the answer is going to be super obvious, but I can't seem to figure it out when looking though docs / examples / discussions.

Basically, I really like the layout, design, and potentially convenient way to use a loop to create multiple Modals using New-HTMLWinBox, but I can't seem to figure out or find any examples of how to attach an event handler to launch them. Both example files for WinBox create them in a way that has all created WinBoxes already open when the page is loaded. I'm looking for an example of how to create a WinBox that starts closed and is only opened when a user clicks on an element (ideally would like be able to attach it to an anchor tag, but would also be okay with modifying the script to use buttons instead).

Really appreciate any advice, examples, or guidance if anyone knows of a solution. Thanks!😊

**Comment by PrzemyslawKlys** [comment]
The bad news is that I just played around with WinBox and never really finished it. The idea was there, including the ability to link it to Calendar, Charts, and Tables, but I never really did go beyond adding the basics. This module is a build in progress and when my projects require it, I try my best to add it. So far I've been able to not require WinBox for my display purposes.

The problem with going with added things like this, the more data you have the slower it gets. So if you display a table and then start adding  stuff on top of it that if you click you get more details, the file size gets out of hand and soon browser struggles.
